backend/controllers/user.controller.ts

```diff
@@ -57,7 +57,7 @@ const resendEmailVerify = async (req: Request, res: Response) => {
     context: {
       subject: "Verification Mail",
       name: user.name,
-      link: `${BASE_URL}api/auth/verifyEmail/${newEmailVerificationToken}/`,
+      link: `${FRONTEND_HOST}api/auth/verifyEmail/${newEmailVerificationToken}/`,
```

Comment:
As we are now using `FRONTEND_HOST` here, You can use `${FRONTEND_HOST}/auth/token/${newEmailVerificationToken}` route which you can see in `frontend/src/routes.tsx`.

**Note:** Currently that token page is used for two token types `"resetPasswordToken", "organizationInvitationToken"`, and we need to implement `newEmailVerificationToken` there in frontend which is seperate task.

Reply:
Just checked the token page its amazing, and easy to organise ❤️